### PR TITLE
[InLong-1012][Doc] Add InLong sort format usage and extend doc

### LINKED ISSUE #1012: [Improve][Doc] Add InLong sort format usage  and extend doc
### Description

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by baomingyu**
Fixes #1012

Add InLong sort format usage and extend doc

### CODE REVIEW

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/development/extension_sort/inlong_sort_data_organization_and_binary_protocol.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+- InLongMsg Tlog-CSV
+- InLongMsg Tlog-KV
+- InLongMsg PB
+- CSV
+- KV
+- JSON
+
+## 开发之前
+
+- InLongMsg 格式介绍参照 [InLongMsg](img/inlong_msg.md)；
```
Comment (aloyszhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/development/extension_sort/inlong_sort_data_organization_and_binary_protocol.md:22]: broken link

File: docs/development/extension_sort/inlong_sort_data_organization_and_binary_protocol.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+- InLongMsg Tlog-KV
+- InLongMsg PB
+- CSV
+- KV
+- JSON
+By implementing the data parsing process for these formats, developers can effectively handle and process data in the InLong-Sort module.
+
+## Before Development
+
+- InLongMsg refer to [InLongMsg](img/inlong_msg.md);
```
Comment (aloyszhang) [inline docs/development/extension_sort/inlong_sort_data_organization_and_binary_protocol.md:25]: broken link

Comment (baomingyu) [reply docs/development/extension_sort/inlong_sort_data_organization_and_binary_protocol.md]: done

**Review by aloyszhang — APPROVED** [review]

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/development/extension_sort/inlong_sort_data_organization_and_binary_protocol.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    * 第三步：通过 DecodingFormat 对象获取 具体格式对应的 DeserializationSchema 对象 （AbstractInLongMsgDeserializationSchema 类的子类）；
+    * 第四步：通过 DeserializationSchema，调用 deserialize(byte\[\] message) 或 deserialize(byte\[\] message, Collector\<T\> out) 方法，传递要解析的数据及获取数据解析后的结果。
+
+  + 扩展流程
+    如图2 所示，扩展 InLongMsg 格式封装的数据格式的解析，需要实现图 2 中的1个接口和继承实现3个基类，图中箭头表示实现间的调用关系：
+
+    ![InLongMsg 格式封装的数据格式解析扩展](img/sort_data_inlongmsg_format_extend.png)
+    <p align="center">图 2 InLongMsg 格式封装的数据格式解析扩展</p>
+    
+    扩ß展流程与图1 中所示流程最大的区别是，中间的获取的 DecodingFormat、DeserializationSchema 分别是 AbstractInLongMsgDecodingFormat 与 AbstractInLongMsgDeserializationSchema 类的子类。
```
Comment (luchunliang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/development/extension_sort/inlong_sort_data_organization_and_binary_protocol.md:55]: Please fix this "扩ß展"

**Review by luchunliang — APPROVED** [review]

**Review by aloyszhang — APPROVED** [review]

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/development/extension_sort/inlong_sort_data_organization_and_binary_protocol.md (hunk 3 above)
Comment (baomingyu) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/development/extension_sort/inlong_sort_data_organization_and_binary_protocol.md]: done

**Review by vernedeng — APPROVED** [review]

**Pull request merged**